scene/resources/navigation_mesh.h

```diff
@@ -84,7 +84,7 @@ class NavigationMesh : public Resource {
 	};
 
 	enum SourceGeometryMode {
-		SOURCE_GEOMETRY_NAVMESH_CHILDREN = 0,
+		SOURCE_GEOMETRY_ROOT_NODE_CHILDREN = 0,
```

Comment:
If this is always the children of the `NavigationRegion`, should this be called `SOURCE_GEOMETRY_NAVIGATION_REGION_CHILDREN`?

Reply:
No because a Navigationmesh resource can be used without any NavigationRegion. You can use the NavigationMeshGenerator singleton directly and point it at any root_node that you want, it does not have to be a NavigationRegion.